### ISSUE TITLE
named: flag uses of unfound identifier

### ISSUE THREAD

**Issue by benmosher** [benmosher/eslint-plugin-import#240]
analogous to namespace references sprinkled into code, i.e.

if

``` js
import { x, z } from 'y' // x is not present in y

function a() {
  return x - z // report x here too
}
```

**Comment by ljharb** [comment]
This seems handled by [import/named](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/named.md).
